internal/node/services/lib/http.go

```diff
@@ -1,14 +1,25 @@
 package lib
 
 import (
+	"encoding/json"
+	"fmt"
 	"log/slog"
+	"net/url"
 	"strings"
 
+	httpclient "github.com/kthomas/go-httpclient"
```

Comment:
Please move this either to an internal module or place all of the code in `internal/node/services/lib/http/*.go`